lib/galaxy/util/miniapp.py

```diff
@@ -0,0 +1,70 @@
+"""Create a smaller version of app for verifying tools"""
+import os
+import shutil
+import tempfile
+from contextlib import contextmanager
+
+from galaxy.datatypes.registry import Registry
+from galaxy.tools.data import ToolDataTableManager
+from galaxy.util.dbkeys import GenomeBuilds
+from .bunch import Bunch
+
+
+class MiniApp(object):
```

Comment:
Entirely personal preference - but what about the path and name ``galaxy.tools.repositories.ValidationContext`` instead of ``galaxy.util.miniapp.MiniApp``?

Reply:
Sure, I can do that!